frontend/tests/Factories/Problem.php

```diff
@@ -67,6 +73,7 @@ public function __construct($params = []) {
         $this->showDiff = $params['show_diff'] ?? 'none';
         $this->allowUserAddTags = $params['allow_user_add_tags'] ?? false;
         $this->problemLevel = $params['problem_level'] ?? 'problemLevelBasicIntroductionToProgramming';
+        $this->publicTags = $params['public_tags'] ?? 'problemTagBinarySearchTree';
```

Comment:
Como aquí ya estamos en PHP, podemos utilizar `json_encode` para codificar el array de objetos en vez de mandar la cadena como en los archivos `.json`:
```suggestion
        $this->publicTags = $params['public_tags'] ?? $params['selected_tags'] ?? json_encode([
            [
                'tagname' => 'problemTagBinarySearchTree',
                'public' => true,
            ],
        ]);
```
Así los puedes mandar en las pruebas donde se utilice directamente `Problem::apiCreate`